src/components/CreateMenu.js

```diff
@@ -7,10 +7,14 @@ import Popover from './Popover';
 import styles from '../styles/styles';
 import themeColors from '../styles/themes/default';
 import Icon from './Icon';
-import {ChatBubble, Users} from './Icon/Expensicons';
+import {
+    ChatBubble, Users, Receipt, MoneyCircle, Paperclip,
+} from './Icon/Expensicons';
 import {redirect} from '../libs/actions/App';
 import ROUTES from '../ROUTES';
 import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import CONST from '../CONST';
```

Comment:
Is it possible to just import `CONST.MENU_ITEM_KEYS` instead of the whole file with React Native?

Reply:
I don't believe we can do this unless  `MENU_ITEM_KEYS` is broken out of the `CONST` object with its own named export.